nixos/modules/services/misc/dendrite.nix

```diff
@@ -222,6 +222,14 @@ in
         for available options with which to populate settings.
       '';
     };
+    openRegistration = lib.mkOption {
+      type = lib.types.bool;
+      default = false;
+      internal = true;
```

Comment:
Is this on purpose? This would make this option not appear on documentation, and is generally used only for options that shouldn't be user setable.

Reply:
If this is really on purpose, please add a comment why.